glean-core/src/upload/directory.rs

```diff
@@ -237,7 +237,7 @@ mod test {
     }
 
     #[test]
-    fn test_creates_requests_correctly_from_valid_ping_file() {
+    fn test_gets_correct_data_from_valid_ping_file() {
```

Comment:
Side-note: (I only noticed that recently again) the `test_` prefix for test functions is not necessary. They are already in a module called `test`, so in the output they show up as `test::test_...` which is kinda double info (no need to change that here now, we have multiple of methods named like that)